### PR TITLE
Changed file section checkbox from <button> to <input type="checkbox">

### PR DESCRIPTION
**Pull request by steve-silverstripe**
BEFORE:
image

AFTER:
image

### CODE REVIEW

**Comment by tractorcow** [comment]
@flashbackzoo @chillu please approve this. :D

File: javascript/src/components/file/index.js [hunk 1]
```diff
@@ -175,29 +175,27 @@ class FileComponent extends SilverStripeComponent {
...
     } else {
-      actionButton = (<button
+      actionInputCheckbox = (<input
         className="item__actions__action item__actions__action--select [ font-icon-tick ]"
-        type="button"
+        type="checkbox"
         title={i18n._t('AssetGalleryField.SELECT')}
         tabIndex="-1"
         onMouseDown={this.preventFocus}
         onClick={this.handleToggleSelect}
```
Comment (flashbackzoo) [inline javascript/src/components/file/index.js]: This should be an `onChange` handler now it's a checkbox. You should also be able to remove the `onMouseDown` handler which looks like it was the as some sort of hack to get around focusing on the button? @scott1702

Comment (scott1702) [inline javascript/src/components/file/index.js]: It was for exactly that @flashbackzoo :grinning:

**Comment by scott1702** [comment]
@steve-silverstripe could you please rebase?

**Comment by clarkepaul** [comment]
I think this might have broken the checkboxes? I think the before and after shots might have been done without a build to the css as when I checked out the latest the markup was still "button" and after a build it was input without the styles.

**Comment by clarkepaul** [comment]
I think its that a input/checkbox cant have padding applied to it so it looses the spacing. I think I'll just swap the padding for margin, should be a quick change. I hope.

**Comment by clarkepaul** [comment]
Input doesn't support ::before in some browsers so they are not visible.

**Pull request merged**